guardian/amigo
# Cloning recipes and base images

## What does this change?

It is frequently the case that a new recipe is needed that is similar to a prior recipe. The same is true of base images. This PR adds the ability to close both of these and also adds the ability to delete unwanted and unused base images (you can already do this for recipes).

## How to test

When deployed to CODE you should be able to clone both recipes and base images and delete both when unused. You should be unable to delete a base image that is used by recipes.

## Images

![image](https://user-images.githubusercontent.com/1236466/101812042-c544c280-3b12-11eb-8185-df0c740044bc.png)
